2020/13xxx/CVE-2020-13663.json

```diff
@@ -1,18 +1,79 @@
 {
```

Comment:
{} is not allowed for {'reference_data': [{'name': 'https://www.drupal.org/sa-core-2020-004', 'refsource': 'CONFIRM', 'url': 'https://www.drupal.org/sa-core-2020-004'}]}